app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/responses/NDRCCaseResponse.scala

```diff
@@ -20,10 +20,11 @@ import play.api.libs.json.{Format, Json}
 
 case class NDRCCaseResponse(
                                        correlationId: String,
+                                       caseId: Option[String],
                                        error: Option[ApiError] = None,
                                        result: Option[NDRCFileTransferResult] = None
```

Comment:
Just checking - is`result: Option[NDRCFileTransferResult]` still needed?

Reply:
It is not - just deleted it and I am running the tests now.